aspnetcore/blazor/index.md

```diff
@@ -57,102 +57,21 @@ Using .NET for client-side web development offers the following advantages:
 > [!NOTE]
 > For a Blazor quick start tutorial, see [Build your first Blazor app](https://dotnet.microsoft.com/learn/aspnet/blazor-tutorial/intro).
 
-## Components
-
-Blazor apps are based on *components*. A component in Blazor is an element of UI, such as a page, dialog, or data entry form.
```

Comment:
I agree that we don't need all of these details about components here that are covered in the Components topic, but I think we still want to say that Blazor apps are made up of components, say why Blazor components are useful (event driven, reusable, etc), and show at least one example of what a Blazor component looks like in code. Devs reading about Blazor for the first time will want to see what writing code in Blazor looks like. Maybe we show just a Counter component?

Reply:
I was leaning too far this way from your remarks on the issue ...

> ... less on how you use it. This doc could probably be a lot shorter and with fewer details that get covered in other docs.

I'll rework this back into the coverage here, shorter with just the `Counter` component.